modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -74,6 +74,12 @@ class {{classname}}(object):
 {{/required}}
 {{/vars}}
 
+{{#allowableValues}}
+    @staticmethod
+    def get_allowed_values():
+        return [{{#enumVars}}{{classname}}.{{name}}{{^-last}}, {{/-last}}{{/enumVars}}]  # noqa: E501
+
+{{/allowableValues}}
```

Comment:
@slash-arun why not add this data as a dict in the python class, not at the instance level?
The data does not change from instance to instance.
See some parallel work on that here:
- template https://github.com/OpenAPITools/openapi-generator/pull/4154/files#diff-03d0471b3e76c94b0460b466312a2332
- result https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/python-experimental/petstore_api/models/enum_test.py

Reply:
Agreed, that makes more sense will push the change, Thanks for reviewing @spacether